src/App.vue

```diff
@@ -302,6 +315,16 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
       this.nameRequestErrorDialog = true
     })
 
+    // listen for invalid delete requests
+    this.$root.$on('delete-error-event', async (error: any) => {
+      console.log('Delete error =', error) // eslint-disable-line no-console
+      this.saveErrors = error?.response?.data?.errors || []
+      this.saveWarnings = error?.response?.data?.warnings || []
```

Comment:
In future ticket, fix these variabes :)

Reply:
Ahh yes 🤦 . Will put in a small commit later today!